libs/dh/market-participant/actors/feature-actors/src/lib/filters/dh-actors-filters.component.ts

```diff
@@ -0,0 +1,155 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  ChangeDetectionStrategy,
+  Component,
+  EventEmitter,
+  Input,
+  OnDestroy,
+  OnInit,
+  Output,
+  inject,
+} from '@angular/core';
+import { FormControl, FormGroup, ReactiveFormsModule } from '@angular/forms';
+import { TranslocoModule, TranslocoService } from '@ngneat/transloco';
+import { Observable, Subject, debounceTime, map, takeUntil } from 'rxjs';
+import { RxPush } from '@rx-angular/template/push';
+
+import { WATT_FORM_FIELD } from '@energinet-datahub/watt/form-field';
+import { WattButtonComponent } from '@energinet-datahub/watt/button';
+import {
+  WattDropdownComponent,
+  WattDropdownOption,
+  WattDropdownOptions,
+} from '@energinet-datahub/watt/dropdown';
+import { MarketParticipantEicFunction } from '@energinet-datahub/dh/shared/domain';
+import { ActorStatus } from '@energinet-datahub/dh/shared/domain/graphql';
+
+import { ActorsFilters } from '../actors-filters';
+
+/** Helper function for creating form control with `nonNullable` based on value. */
+const makeFormControl = <T>(value: T | null = null) =>
```

Comment:
Question: I´m pretty sure a saw this in one of Bjørns PR´s as well. Maybe we should move it to a helper?

Reply:
Yes, it's the same helper function. Will move it to a shared lib in a follow up PR.